companion/src/eepromimportexport.h

```diff
@@ -698,8 +698,11 @@ class ConversionField: public TransformedField {
       if (table) {
         if (table->exportValue(_field, _field))
           return;
-        if (!error.isEmpty())
+        if (!error.isEmpty()) {
+          // TODO how could we avoid this global?
```

Comment:
Not sure there's a quick fix here... who "owns" `EEPROMWarnings`?  

Basically to avoid globals we need everything to belong to a class, even if that class is a singleton.  Then use singletons as, eg. (totally made up):
```
EepromInterface myEepe = EepromInterface::instance();
QStringList warns = myEeepe.warnings();  // or whatever we want to do with the instance
```